crypto/src/error.rs

```diff
@@ -28,6 +28,10 @@ quick_error! {
             cause(e)
             from()
         }
+        ZeroIterations {
+            description("Iterations' value should not be zero")
+            from()
```

Comment:
I think this `from()` does nothing. Would you check?

Reply:
The code works without from(), I removed it.